rts/lra/test/basic/src/test/resources/logging.properties

```diff
@@ -15,9 +15,10 @@
 #
 
 handlers = java.util.logging.ConsoleHandler
-java.util.logging.ConsoleHandler.level = INFO
+java.util.logging.ConsoleHandler.level = FINEST
```

Comment:
Why finest by default - can't we add that in the jobs or on demand if an engineer is debugging a particular issue. This adds a lot of noise, slows down running local builds and makes it impossible to quickly scan the output looking for failures. I see some profiles in the top level pom for enabling trace level logging - how does that compare to this properties file.